src/shims/@zimbra-client/hooks/graphql/index.js

```diff
@@ -12,5 +12,6 @@ export const useAccountInfo = wrap('useAccountInfo');
 export const useDocumentAction = wrap('useDocumentAction');
 export const useBriefcaseFoldersQuery = wrap('useBriefcaseFoldersQuery');
 export const useSaveDocumentMutation = wrap('useSaveDocumentMutation');
+export const useSearchGalLazyQuery = wrap('useSearchGalLazyQuery');
```

Comment:
I thought we have moved to using fetch directly for soap endpoint instead of going through graphql, do we need this now?